.circleci/scripts/upload_cloud_watch_to_s3.py

```diff
@@ -0,0 +1,50 @@
+import argparse
+import time
+import sys
+
+import boto3
+
+logs = boto3.client("logs")
+
+
+def upload_cloud_watch_logs(log_group_name: str, bucket_name: str, prefix: str) -> str:
+    task_id = logs.create_export_task(
+        logGroupName=log_group_name,
+        fromTime=0,
+        to=int(round(time.time() * 1000)),
+        destination=bucket_name,
+        destinationPrefix=prefix
+    )["taskId"]
+
+    print(f"Uploading logs to s3://{bucket_name}/{prefix}/{task_id}.")
+    return task_id
+
+
+def wait_for_export_task(task_id: str) -> None:
+    while True:
```

Comment:
blocking: let's not loop forever -- this is bounded to 5 hours but that's still a long time

Reply:
Good idea, added a 5 min timeout for this.